src/scores/probability/crps_impl.py

```diff
@@ -786,19 +786,19 @@ def crps_for_ensemble(
 
 
     .. math::
-        CRPS(x_i, y) = (1 / M) * sum(|x_i - y|) - (1 / 2 * K) * sum(|x_i - x_j|)
+        CRPS(x_i, y) = \\frac{sum(|x_i - y|)}{M} - \\frac{sum(|x_i - x_j|)}{2K}
```

Comment:
Need to change `sum` to `\\sum_{}^{}`
```suggestion
        CRPS(x_i, y) = \\frac{\\sum_{i=1}^{M}(|x_i - y|)}{M} - \\frac{\\sum_{i=1}^{M}(|x_i - x_j|)}{2K}
```